index.html

```diff
@@ -6,14 +6,15 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Portifolio setup and mobile first</title>
     <link rel="stylesheet" href="style.css">
+		<link rel="stylesheet" href="desktop.css">
 </head>
 <body>
 	<header>
 		<div class="logo">Patrick Wanjiru</div>
 		<ul>
-			<li><a href=""></a></li>
-			<li><a href=""></a></li>
-			<li><a href=""></a></li>
+			<li><a href="">Portifolio</a></li>
+			<li><a href="">Contact</a></li>
+			<li><a href="">About</a></li>
```

Comment:
- [ ] [**OPTIONAL**] Good job implementing these navlist :clap: , however you should make them more responsive, so they bring the user to corresponding section while scrolling :call_me_hand: 

![Screenshot from 2023-01-21 02-00-48](https://user-images.githubusercontent.com/80458199/213827258-573aef9c-84ae-4ee6-8dd8-1a36991c7cd1.png)